src/manual/integrations/hooks.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Hooks
+order: 50
+---
+
+The [Hooks service](/reference/core/hooks/) is responsible for creating
```

Comment:
There's  more detail there -- as a manual page, I want to outline generally how / when someone might use the hooks service.  It's not too impressive right now because it only does timed tasks.